docker/types/__init__.py

```diff
@@ -1,5 +1,6 @@
 # flake8: noqa
-from .containers import ContainerConfig, HostConfig, LogConfig, Ulimit
+from .containers import \
+    ContainerConfig, HostConfig, LogConfig, Ulimit, DeviceRequest
```

Comment:
Why not use `(...)` (as below and in other files) instead of `\`?

Reply:
Not much of a reason really; I just tend to use `\` when there is one new line and `(…)` when there are more. Changed it in fd20b24